autoyast/provision_sles.erb

```diff
@@ -124,6 +124,7 @@ cp /etc/resolv.conf /mnt/etc
 <% if salt_enabled %>
 <%= snippet 'saltstack_setup' %>
 <% end -%>
+
 <% if spacewalk_enabled -%>
```

Comment:
For SLES that's only to register the system, main reason being that we add puppet/salt repos and packages in the first install step in the XML anyway. I do get the point, but if this needs to be changed that would probably require refactoring which can only be done when having such a setup available for testing.